config/schema_config.yaml

```diff
@@ -773,6 +777,7 @@ promoter to gene association:
   properties:
     score: float
     biological_context: str
+    distance: int
```

Comment:
does it make sense to add `distance` as a property? It can be calculated dynamically on the fly